src/Assert.php

```diff
@@ -404,7 +410,50 @@ public static function isInstanceOfAny($value, array $classes, $message = '')
         }
 
         static::reportInvalidArgument(sprintf(
-            $message ?: 'Expected an instance of any of %2$s. Got: %s',
+            $message ?: 'Expected an instance of this class or to this class among his parents of any of %2$s. Got: %s',
+            static::typeToString($value),
+            implode(', ', array_map(array('static', 'valueToString'), $classes))
+        ));
+    }
+
+    public static function isAOf($value, $class, $message = '')
+    {
+        static::string($class, 'Expected class as a string. Got: %s');
+
+        if (!is_a($value, $class, is_string($value))) {
+            static::reportInvalidArgument(sprintf(
+                $message ?: 'Expected an instance of this class or to this class among his parents %2$s. Got: %s',
+                static::typeToString($value),
+                $class
+            ));
+        }
+    }
+
+    public static function notAOf($value, $class, $message = '')
```

Comment:
Maybe the name `isNotA` would be more appropriate here, what do you think?

Reply:
I used the same naming convention as it is used by `instanceOf`. If you prefer your suggested names I will change them. :-)

Reply:
Renamed as proposed